composer.lock

```diff
@@ -4,7 +4,8 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
         "This file is @generated automatically"
     ],
-    "content-hash": "0ceb266d9819ca0d12f1388267bc7375",
+    "hash": "de7fb5224e4a1eb6ae01d38acb37c023",
+    "content-hash": "d171c3ed89ae5dc94390939ef9f1514e",
```

Comment:
A lot of these changes are due to using an old version of Composer. Redo this after a `composer self-update`.